lib/cronofy/auth.rb

```diff
@@ -22,8 +23,17 @@ def get_token_from_code(code, redirect_uri)
       @client.auth_code.get_token(code, :redirect_uri => redirect_uri).token
     end
 
-    def request
-      @request ||= OAuth2::AccessToken.new(@client, @token)
+    # Public: Refreshes the access token
+    # Returns Hash of token elements to allow client to update in local store for user
+    def refresh!
+      @access_token = access_token.refresh!
```

Comment:
I was thinking of keeping it consistent with what was returned by the API http://www.cronofy.com/developers/api#token-issue. Agree that the retrieval of the original token should do the same.